src/Nethermind/Nethermind.Mev/MevPlugin.cs

```diff
@@ -157,16 +159,30 @@ public async Task<IBlockProducer> InitBlockProducer(IConsensusPlugin consensusPl
                 _nethermindApi.LogManager);
 
             _nethermindApi.BlockProducerEnvFactory = producerEnvFactory;
+            
+            if (_mevConfig.MaxMergedBundles == null || _mevConfig.MaxMergedBundles < 1)
+            { 
+                throw new ArgumentException("MaxMergedBundles cannot be null or zero");
```

Comment:
nameof(_mevConfig.MaxMergedBundles)

Not sure if we want to throw here...

Reply:
we could make it accept 0 and then only produce vanilla blocks